File: Atividade_contextualizacao/ServerPython/Aluno.py
class Aluno:
    def __init__(self,id,nome,matriculado):
        self.id = id
        self.nome = nome
        self.matriculado = matriculado
    
    def getId(self):
        return self.id

    def setId(self, id):
        self.id = id
    
    def getNome(self):
        return self.nome

    def setNome(self, nome):
        self.nome = nome
    
    def getMatriculado(self):
        return self.matriculado
    
    def setMatriculado(self, matriculado):
        self.matriculado = matriculado

    def __srt__(self):
        return "{} {} {}".format(self.id, self.nome, self.matriculado)

File: Atividade_contextualizacao/ClienteJava/src/tsi/webservices/comunicacaoSocket/modelos/Aluno.java
package tsi.webservices.comunicacaoSocket.modelos;

public class Aluno {

	private static int id = 0;

	private int idAluno;
	private String nome;
	private int matriculado;

	public Aluno(String nome, int matriculado) {
		id++;
		this.idAluno = id;
		this.nome = nome;
		this.matriculado = matriculado;
	}

	public int getId() {
		return idAluno;
	}

	public void setId(int idAluno) {
		this.idAluno = idAluno;
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public int isMatriculado() {
		return matriculado;
	}

	public void setMatriculado(int matriculado) {
		this.matriculado = matriculado;
	}

	@Override
	public String toString() {
		return idAluno + " " + nome + " " + matriculado;
	}
}
